psi4/src/psi4/libqt/qt.h

```diff
@@ -135,6 +135,7 @@ void C_DTRMM(char side, char uplo, char transa, char diag, int m, int n, double
 void C_DSYRK(char uplo, char trans, int n, int k, double alpha, double* a, int lda, double beta, double* c, int ldc);
 void C_DSYR2K(char uplo, char trans, int n, int k, double alpha, double* a, int lda, double* b, int ldb, double beta,
               double* c, int ldc);
+PSI_API
```

Comment:
For the record, can we get a comment saying why this was added to the API?

Reply:
Actually I do not know why only certain functions are available via the API. Anyway, I revise the generalized minimal residual algorithm in forte and one of the steps is to solve a triangular system.